idb/postgres/postgres.go

```diff
@@ -2222,10 +2237,19 @@ func (db *IndexerDb) yieldAccountsThread(req *getAccountsRequest) {
 					NumUint:      ls[i].Schema.NumUint,
 				}
 				aout[i].KeyValue = ls[i].KeyValue.toModel()
+				if aout[i].Deleted == nil || !*aout[i].Deleted {
+					totalSchema.NumByteSlice += ls[i].Schema.NumByteSlice
+					totalSchema.NumUint += ls[i].Schema.NumUint
+				}
+
 			}
 			account.AppsLocalState = &aout
 		}
 
+		if totalSchema.NumByteSlice != 0 || totalSchema.NumUint != 0 {
```

Comment:
probably better to write `totalSchema != models.ApplicationStateSchema{}`

Reply:
done